examples/asg_inital_lifecycle_hook/README.md

```diff
@@ -0,0 +1,28 @@
+# Auto Scaling Group without ELB example
```

Comment:
"Auto Scaling Group with initial lifecycle hook"

Reply:
Also, add a link to this example in the main README.md file.

Reply:
@antonbabenko I'm away at the moment. I will try and get these issues sorted.